text/3128-io-safety.md

```diff
@@ -417,21 +417,21 @@ patiently answering my many questions!
 [@RalfJung]: https://github.com/RalfJung
 [`File`]: https://doc.rust-lang.org/stable/std/fs/struct.File.html
 [`TcpStream`]: https://doc.rust-lang.org/stable/std/net/struct.TcpStream.html
-[`FromRawFd`]: https://doc.rust-lang.org/stable/std/os/unix/io/trait.FromRawFd.html
+[`FromRawFd`]: https://doc.rust-lang.org/std/os/fd/trait.FromRawFd.html
```

Comment:
For parity with the other links here, the `stable/` part of the path should probably be kept:

```suggestion
[`FromRawFd`]: https://doc.rust-lang.org/stable/std/os/fd/trait.FromRawFd.html
```

Reply:
Nice, let me apply this change!

Reply:
done